site/src/pages/RoadmapPage/ImportTranscriptPopup.tsx

```diff
@@ -7,7 +7,7 @@ import { useAppDispatch } from '../../store/hooks';
 import { parse as parseHTML, HTMLElement } from 'node-html-parser';
 import ThemeContext from '../../style/theme-context';
 import { BatchCourseData, PlannerQuarterData, PlannerYearData } from '../../types/types';
-import { normalizeQuarterName, quarterNames } from '../../helpers/planner';
+import { quarters as quarterNames } from '@peterportal/types';
```

Comment:
should we consider renaming this to `quarterNames` in `@peterportal/types` altogether since it's just names and not JSON?

Reply:
Yeah, the type actually comes from peterportal-api-next-types though which is exported by `@peterportal/types`

Reply:
I see. In that case, we might want to rename the export

Reply:
Added an export for `quarterNames`